spec/factories/admin_users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :admin_user do
+    email { [Faker::Internet.email].sample }
```

Comment:
Sorry, I meant the other one - `sequence :email { |n| "#{n}#{Faker::Internet.email}" }`. Another option is to get rid of Faker completely here: `sequence :email { |n| "admin#{n}@example.com" }`.

The `sequence` is the key thing here: we've found that e.g. `Faker::Internet.email` can sometimes return the same value on consecutive calls. So, if a test file tries to create two AdminUsers and they both get the same Faker email, the second record will be invalid since its email won't be unique. Appending a sequential number ensures that even if Faker returns e.g. "abc@this.com" twice, the final email won't be the same: "1abc@this.com", "2abc@this.com" etc.